core/object/object.cpp

```diff
@@ -738,23 +738,28 @@ void Object::setvar(const Variant &p_key, const Variant &p_value, bool *r_valid)
 }
 
 Variant Object::callv(const StringName &p_method, const Array &p_args) {
-	const Variant **argptrs = nullptr;
+    const Variant **argptrs = nullptr;
```

Comment:
Please use tabs for indentation. Ideally, your IDE should do this for you :)

Reply:
Hi i fixed this issue and uptaded the commit

Reply:
You need to fix it in variant_call.cpp as well, it looks like.

Reply:
Uptaded.

Reply:
Hi, please check the "Files Changed" page to see what your changes actually are. There's still a bunch of incorrect indentation: https://github.com/godotengine/godot/pull/100215/files

Reply:
Can you approve sir